include/openssl/ssl.h

```diff
@@ -169,9 +169,13 @@ extern "C" {
  * The following cipher list is used by default. It also is substituted when
  * an application-defined cipher list string starts with 'DEFAULT'.
  * This applies to ciphersuites for TLSv1.2 and below.
+ * DEPRECATED IN 3.0.0, in favor of OSSL_default_cipher_list()
```

Comment:
I think you want `#if OPENSSL_API_LEVEL < 4`, but I'm not really sure.

@levitte: Can you comment on how to do this properly?

Reply:
You are correct

Reply:
So, wrap the `#define`s with that `#ifdef`?

Reply:
Yes